libraries/provider_configure.rb

```diff
@@ -91,14 +103,21 @@ def action_manage
     params[:MAX_MAP_COUNT] = new_resource.max_map_count
     params[:ES_JVM_OPTIONS] = "#{new_resource.path_conf}/jvm.options"
 
+    params = params.merge(new_resource.environments) if new_resource.environments
+
+    parameters = {}
+    params.each do |key, value|
+      parameters[key.to_sym] = value
+    end
+
     default_config_name = es_svc.service_name || es_svc.instance_name || new_resource.instance_name || 'elasticsearch'
 
     shell_template = template "elasticsearch.in.sh-#{default_config_name}" do
       path node['platform_family'] == 'rhel' ? "/etc/sysconfig/#{default_config_name}" : "/etc/default/#{default_config_name}"
       source new_resource.template_elasticsearch_env
       cookbook new_resource.cookbook_elasticsearch_env
       mode '0644'
-      variables(params: params)
+      variables(params: parameters)
```

Comment:
Hi there -- you're shipping some new parameters but there's no update to the template. Did you mean to include the template in this PR?